packages/apollos-api/src/data/content-items/data-source.js

```diff
@@ -230,6 +221,69 @@ class ExtendedContentItem extends ContentItem.dataSource {
     return [mostRecentSermon];
   };
 
+  async getSeriesWithUserProgress() {
```

Comment:
What do we have on core? How does this differ? Did this just not get built on core ever?

Reply:
Core works slightly differently, and I think core should be updated to the logic here. I made this realization while testing, and so had to write this code. 

Core says - if I have completed 100% of a series, don't show it. 
Willow says - if I have completed the most recent entry of a series, don't show it. 

The problem is that the "Continue" button won't show up if you have completed 50% of a series, but have completed the most recent entry.

Reply:
It's only that "getLastChildComplete" method call on line 255 that changes.